appflavors/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="com.bugsnag.appflavors">
+
+  <application
+    android:allowBackup="true"
+    android:label="App">
+    <activity android:name=".MainActivity">
+      <intent-filter>
+        <action android:name="android.intent.action.MAIN" />
+
+        <category android:name="android.intent.category.LAUNCHER" />
+      </intent-filter>
+    </activity>
+
+    <meta-data
+      android:name="com.bugsnag.android.API_KEY"
+      android:value="fake-api-key" />
```

Comment:
I think standard has been `YOUR_API_KEY` among the examples I've seen, but that's personal preference.